A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -994,11 +994,11 @@ class CDUPerformancePage {
             bottomRowCells,
         ]);
     }
-    static UpdateThrRedAccFromOrigin(mcdu, updateThrRedAlt = true, updateAccAlt = true) {
+    static UpdateThrRedAccFromOrigin(mcdu, updateThrRedAlt = true, updateAccAlt = true, autoRefresh = false) {
```

Comment:
Hm, this is a bit different than it currently works. Right the it's the responsibility of the thing calling `UpdateThrRedAccFromOrigin` to ensure it doesn't overwrite pilot input, hence `updateThrRedAlt` and `updateAccAlt`. If you move this check inside the component you don't really need to specify what you're updating, you can just call this whenever.

Reply:
I don't really understand what you are trying to say with ` you don't really need to specify what you're updating, you can just call this whenever.`